src/unity/python/turicreate/visualization/plot.py

```diff
@@ -0,0 +1,29 @@
+import time
+import json
+
+class Plot(object):
```

Comment:
I was imagining that this class would inherit from `tc.extensions._Plot`, instead of containing it as a member, but there may be reasons for that not to make sense. If it works better as a member that's fine.